pyls/plugins/flake8_lint.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2017 Palantir Technologies, Inc.
+import logging
+from flake8.api import legacy as flake8
+from pyls import hookimpl, lsp
+
+log = logging.getLogger(__name__)
+
+
+@hookimpl
+def pyls_lint(config, document):
+    settings = config.plugin_settings('flake8')
+    log.debug("Got flake8 settings: %s", settings)
+
+    opts = {
+        'exclude': settings.get('exclude'),
+        'filename': settings.get('filename'),
+        'hang_closing': settings.get('hangClosing'),
```

Comment:
what's the deal with camelCase vs snake_case here?

Reply:
I got that part from another plugin using the same conf, but it looks like it wasn't working, so I'm willing to debug this to fix it

Reply:
VSCode settings convention appears to be camel case, but we need to pass it to the Flake8 as a snake case option

Reply:
@gatesn how does it affects VSCode if for example we rename plugins.pycodestyle.hangClosing to plugins.pycodestyle.hang-closing ?

Reply:
Renaming configs would likely break existing users. We could build utility functions to convert between case styles, but not sure it's that bad as-is

Reply:
got it